cmd/go-ycsb/client.go

```diff
@@ -42,6 +42,10 @@ func runClientCommandFunc(cmd *cobra.Command, args []string, doTransactions bool
 		if cmd.Flags().Changed("target") {
 			globalProps.Set(prop.Target, strconv.Itoa(targetArg))
 		}
+
+		if cmd.Flags().Changed("keyprefix") {
```

Comment:
it is not a widely used option, can you use a property instead, I suggest not using this in command arguments.

Reply:
Done. Removed it from command arguments.